### PR TITLE
interpreter: use interpreter default bitwidth in run main

### PR DESCRIPTION
**Pull request by superlopuh**

### CODE REVIEW

File: xdsl/tools/xdsl_run.py [hunk 1]
```diff
@@ -62,6 +62,7 @@ def register_all_arguments(self, arg_parser: argparse.ArgumentParser):
         arg_parser.add_argument(
             "--index-bitwidth",
             choices=(32, 64),
+            default=Interpreter.DEFAULT_BITWIDTH,
```
Comment (alexarice) [inline xdsl/tools/xdsl_run.py:65]: This is defined as the system bitwidth anyway

Comment (alexarice) [reply xdsl/tools/xdsl_run.py:65]: Not clear to me what happened before if this wasn't specified

**Comment by superlopuh** [comment]
I'll look into this and try to determine the difference in behaviour

File: xdsl/tools/xdsl_run.py (hunk 1 above)
Comment (superlopuh) [reply xdsl/tools/xdsl_run.py:66]: I added a comment, the argument value is None by default, so it was assigning None explicitly in the init, which was propagated

**Review by alexarice — APPROVED** [review]

**Pull request merged**